mi/dataset/driver/spkir_abj/dcl/resource/rec_20020113.spkir2.yml

```diff
@@ -1,200 +1,186 @@
 header:
-  particle_object: MULTIPLE
-  particle_type: MULTIPLE
-
+    particle_object: 'MULTIPLE'
+    particle_type: 'MULTIPLE'
 data:
-  - _index:  1
+  - _index: 1
     particle_object: SpkirAbjDclRecoveredInstrumentDataParticle
     particle_type: spkir_abj_dcl_instrument_recovered
-    internal_timestamp: '2002-01-13T03:07:07.338Z'
-    dcl_controller_timestamp: '2002/01/13 03:07:07.338'
-    instrument_id: 'SATDI1'
-    serial_number: '0001'
-    timer:       0.00
-    sample_delay: -32768
+    port_timestamp: 3219880027.3380003
```

Comment:
one date string for port_timestamp and other date string for instrument time, so i removed both date string from value and set them as port_timestamp and internal_timestamp in ntp format